cli/templates/Python/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.8.3-alpine3.10
```

Comment:
python:3.8.3 is the minimal version for `borsh-construct` (required for protectedData use cases)